src/uuid-options.ts

```diff
@@ -0,0 +1,6 @@
+export interface UuidOptions {
```

Comment:
Doesn't really need it's own file, you could keep it at the top of `secure-dfu.ts`?